python-checks/src/test/resources/checks/functionReturnTypeWithConditionalImport.py

```diff
@@ -0,0 +1,8 @@
+import sys
```

Comment:
I think all this could go into a function in the other test file, so we keep only one extra test file (if you decide to do this, it may make sense to rename the test file then to broaden it from just `collections.abc`.